modules/management-console-agent/src/main/java/org/apache/ignite/agent/utils/AgentUtils.java

```diff
@@ -64,7 +64,7 @@ private AgentUtils() {
      * @param clusterId Cluster ID.
      */
     public static String monitoringUri(String srvUri, UUID clusterId) {
-        return srvUri + "/clusters/" + clusterId + "/monitoring-dashboard";
+        return srvUri.replaceAll("/+$", "") + "/clusters/" + clusterId + "/monitoring-dashboard";
```

Comment:
Could we have utility function that will be the same signature as `java.nio.file.Paths#get(java.lang.String, java.lang.String...)` ?
For example: `Utility.buildUri(String first, Object... more)` ? And hide all logic for `\` manipulation in utility function?
So usage will be: `buildUri(srvUri, "clusters", clusterId,  "monitoring-dashboard");`

Reply:
Try something like this or search in spring utils.
```
URI uri = URI.create(srvUri);

try {
    uri = new URI("http".equalsIgnoreCase(uri.getScheme()) ? "ws" : "wss",
        uri.getUserInfo(),
        uri.getHost(),
        uri.getPort(),
        "/clusters/" + clusterId + "/monitoring-dashboard",
        null,
        null
    );
}
catch (URISyntaxException x) {
    throw new IllegalArgumentException(x.getMessage(), x);
}
```

Reply:
Replace on StringUtils#trimTrailingCharacter